moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp

```diff
@@ -372,11 +372,11 @@ ompl_interface::ModelBasedPlanningContextPtr ompl_interface::PlanningContextMana
     else
     {
       // Choose the correct simple setup type to load
-      context_spec.ompl_simple_setup_.reset(new ompl::geometric::SimpleSetup(context_spec.state_space_));
+      context_spec.ompl_simple_setup_ = std::make_shared<ompl::geometric::SimpleSetup>(context_spec.state_space_);
     }
 
     ROS_DEBUG_NAMED(LOGNAME, "Creating new planning context");
-    context.reset(new ModelBasedPlanningContext(config.name, context_spec));
+    context_spec.ompl_simple_setup_ = std::make_shared<ompl::geometric::SimpleSetup>(context_spec.state_space_);
```

Comment:
the modernization with std::make_shared introduced a regression. With these two line changes I cannot run the Constrained planning tutorial anymore, it is crashing.

Reply:
:( I remember similar problems last year. Do you have some error messages?

Reply:
unfortunately it is just crashing.

Reply:
Jup this was clearly a bug, my mistake :s I ran the tutorial locally again now and it seems to be working again. (Without the support for orientation constraints with this planner of course, but it just falls back to the PoseModel planner.) (FWI turning on debug messages can be really helpful http://dav.ee/blog/notes/archives/898 :) )